src/models/PMCMean.cpp

```diff
@@ -0,0 +1,145 @@
+#include "PMCMean.h"
+#include "../constants.h"
+#include <cmath>
+#include <cstdio>
+#include "../doctest.h"
+
+int Pmc_mean::fit_value_pmc(float value, int is_error_absolute){
+    float next_min_value = min_value < value ? min_value : value;
```

Comment:
Nok bedst at have is absolute med i constructor, giver ikke rigtigt mening at sende med hver gang.

Reply:
fixed